File: CRS-Group-A-Java-Spring-REST/src/main/java/com/lti/bean/Grade.java
package com.lti.bean;

import org.springframework.stereotype.Component;

/**
 * @author Sebastian
 *
 */

@Component
public class Grade {
	
	private String grade;
	private Course course;
	
	public Grade(String grade, Course course) {
		
		this.grade = grade;
		this.course = course;
	}

	public String getGrade() {
		return grade;
	}

	public void setGrade(String grade) {
		this.grade = grade;
	}

	public Course getCourse() {
		return course;
	}

	public void setCourse(Course course) {
		this.course = course;
	}
	
	
}

File: CRS-GROUP-A-Angular-UI-UX-App/src/app/models/grade.ts
import { Course } from "./Course";

export class Grade{
	
	private grade: string;
	private course: Course;
	
	constructor(grade: string, course: Course) {
		
		this.grade = grade;
		this.course = course;
	}

	getGrade(): string {
		return this.grade;
	}

	setGrade(grade: string) {
		this.grade = grade;
	}

	getCourse(): Course {
		return this.course;
	}

	setCourse(course: Course) {
		this.course = course;
	}
	
	
}